examples/apps/cryptoexample/README

```diff
@@ -0,0 +1,38 @@
+sailfishcryptoexample
+
+This is a simple application which stores a secret into an
+application-specific collection (wallet) which is protected
+by the device lock.
+
+That secret is then used to generate a symmetric key,
+which is used either to encrypt $CWD/plaintext.dat
+and write the output to $CWD/ciphertext.dat, or to
+decrypt $CWD/ciphertext.dat and write the output to
+$CWD/plaintext.dat.
+
+--
+
+Some notes on the implementation:
+
+- By using OwnerOnlyMode access control mode, the secrets
+  daemon will not allow any other application to access
+  the collection of secrets.
+
+- By using DeviceLock protection, the secrets daemon doesn't
+  need to ask the user for the lock code to unlock the
+  collection when the application attempts to access it,
+  but will ask for permission if required (e.g. after reboot).
```

Comment:
This has the disadvantage that the user will be presented a system dialog to let the application access the secret, the next time the application is run after a reboot.  This behaviour (always require user to explicitly unlock collections) was requested by one of our partners, but it might be useful to add an "ApplicationData" mode which doesn't require any user input.

(This mode can currently be emulated by registering an application-specific InteractionView which always returns the same key, instead of actually prompting the user for input, however that is obviously a suboptimal workaround.)